pkg/plugin/plugin.go

```diff
@@ -656,7 +656,7 @@ func (e *AutoscaleEnforcer) Score(
 
 	// Special case: return minimum score if we don't have room
 	overbudget, verdict := e.speculativeReserve(node, vmInfo, pod, false, func(_ verdictSet, overBudget bool) bool {
-		return overBudget
+		return false // never actually accept the pod; we're just doing this to ask if it's over-budget.
```

Comment:
How we could've prevented it in theory? I see two ways which could've helped:
1. Passing the arguments (and especially callbacks) through the struct, thus giving it a name on the call site, increasing the chance one of us would spot the bug.
2. Having a custom return type (wrapper over bool).

Reply:
Yeah, I was thinking that as well... Good to know going forward w.r.t. avoiding plain bools in return types for things like this where the meaning isn't obvious from context.